### PR TITLE
[SCSE-187] Override Chakra UI Button base styles

### PR DESCRIPTION
**Pull request by jack-thant**
Please review my code, thank you

[SCSE-187]
[SCSE-223]

### CODE REVIEW

**Review by jack-thant — COMMENTED** [review]
Please review my code, thanks

**Comment by jack-thant** [comment]
I have fixed the button link error and standardised button link across the components and pages

File: apps/web/pages/academics.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { Button, FooterContentText, FooterContentTextProps, Hero } from "ui";
+import { ButtonLink as Button, FooterContentText, FooterContentTextProps, Hero } from "ui";
```
Comment (xJQx) [inline apps/web/pages/academics.tsx:1]: Don't need to rename to 'Button'. Can keep it as 'ButtonLink'.

**Review by xJQx — CHANGES_REQUESTED** [review]

File: packages/ui/components/button/ButtonLink.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    label: string
+    href: string
+    variant?: 'primary-blue' | 'primary-black'
+}
+
+export const ButtonLink = (props: ButtonLinkProps) => {
+  const { label, href,  variant='primary-blue', ...buttonProps } = props
+    return (
+
+      <Link href={ href } _hover={{ textDecoration: 'none' }}>
```
Comment (xJQx) [inline packages/ui/components/button/ButtonLink.tsx:15]: The ButtonLink style has changed after shifting the Link component here. Please update the styling. Thanks!

**Review by xJQx — CHANGES_REQUESTED** [review]

**Pull request closed without merging**